woodwork/logical_types.py

```diff
@@ -589,6 +589,18 @@ class PhoneNumber(LogicalType):
 
     primary_dtype = "string"
 
+    def validate(self, series, return_invalid_values=False):
+        """Validates PhoneNumber values based on the regex in the config.
```

Comment:
Can we put something in the docstring that by default, we use a regex for North American Phone Numbers?